packages/admin/cms-admin/src/blocks/createSeoBlock.tsx

```diff
@@ -252,6 +268,71 @@ export function createSeoBlock({ image = PixelImageBlock }: CreateSeoBlockOption
                             </Typography>
                             <Field label={<FormattedMessage {...messages.url} />} name="canonicalUrl" component={FinalFormInput} fullWidth />
                         </Box>
+
+                        {/* Alternate Hreflang */}
+                        <Box marginTop={8} marginBottom={8}>
+                            <AdminComponentSectionGroup
+                                title={<FormattedMessage id="comet.blocks.seo.alternativeLinks.sectionTitle" defaultMessage="Alternate Hreflang" />}
+                            >
+                                <AdminComponentPaper>
+                                    <FieldArray name="alternativeLinks">
+                                        {({ fields }) => (
+                                            <>
+                                                {fields.map((link, i) => (
+                                                    <Grid key={i} container spacing={2} sx={{ marginBottom: 2 }}>
+                                                        <Grid item xs={3}>
+                                                            <Field
+                                                                label={
+                                                                    <FormattedMessage
+                                                                        id="comet.blocks.seo.alternativeLinks.code"
+                                                                        defaultMessage="Code"
+                                                                    />
+                                                                }
+                                                                name={`${link}.code`}
+                                                                component={FinalFormInput}
+                                                                placeholder="en-US"
+                                                            />
+                                                        </Grid>
+                                                        <Grid item xs>
+                                                            <Field
+                                                                label={<FormattedMessage {...messages.url} />}
+                                                                name={`${link}.url`}
+                                                                component={FinalFormInput}
+                                                                validate={(url) => {
```

Comment:
Please extract this validator into a generic `validateUrl` validator that is used by both the SEO block and the external link block (And change the ID of the message accordingliy)